sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/experiment_events_live_v1/view.sql

```diff
@@ -0,0 +1,64 @@
+CREATE MATERIALIZED VIEW org_mozilla_fenix_derived.experiment_events_live_v1
```

Comment:
I'm not sure what the right destination datasets for these materialized are. We will have materialized views which are almost identical for the following datasets:
* `org_mozilla_fenix_live`
* `org_mozilla_fenix_nightly_live`
* `org_mozilla_firefox_beta_live`
* `org_mozilla_fennec_aurora_live`
* `org_mozilla_firefox_live`

and I'm not sure whether all of them should live in `telemetry_derived` or in the Glean dataset.

The reason for having separate materialized views is that previously [a `UNION ALL`](https://github.com/mozilla/bigquery-etl/blob/9b1d4091857cec6d091c60e671afa26dbd1694be/sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_base/view.sql#L16) was done to get live events from all Fenix-related datasets. However, materialized views do not support `UNION ALL`.